### ISSUE TITLE
A little suggestion

### ISSUE THREAD

**Issue by RyuuZaky** [awshawka/react-native-double-tap#2]
I would recommend to use `TouchableWithoutFeedback` instead of `TouchableOpacity`... in my case, I had to modify it because i didn't want it to show that opacity change when the user taps the container. 

Thanks for this man, works good. 👍

**Comment by Dellybro** [comment]
I agree, i really wanted to use this component, but i had to use my own because you can't pass activeOpacity as a prop.

**Comment by saurabhkanswal** [comment]
> I would recommend to use `TouchableWithoutFeedback` instead of `TouchableOpacity`... in my case, I had to modify it because i didn't want it to show that opacity change when the user taps the container.
> 
> Thanks for this man, works good. 👍

Just in case if  anyone is looking to achieve this, then just change `TouchableWithoutFeedback` instead of `TouchableOpacity` inside node_modules\react-native-double-tap\index.js
